README.md

```diff
@@ -6,46 +6,288 @@
+this is what Git Machete displays when `toggle listing commits` is enabled.
+
+If one of branch layout edges is yellow, that means the fork point of this branch cannot be unambiguously determined.
+However, Git Machete can give you a hint which commit is probably the fork point.
+Show commits to see the suggested fork point (see the `Toggle listing commits` section).
+
+Now you can use `Override Fork Point of Branch` action to choose the fork point of this branch.
+It can be commit inferred by Git Machete (the one marked in commits list), or the one that parent branch is pointing to.
+
+![](docs/override_forkpoint.gif)
+
+</details>
+
+<details>
+<summary><strong>Other actions</strong></summary>
+<br/>
+
+On the left side bar you can find other actions (from top to bottom):
+- ![](docs/left_bar_actions/refresh.png) **Refresh Status** &mdash; refresh the graph displayed in main plugin window
+- ![](docs/left_bar_actions/toggle_listing_commits.png) **Toggle Listing Commits** &mdash; show or hide commits belonging to branches (for more details see `Toggle listing commits` section)
+- ![](docs/left_bar_actions/open_machete_file.png) **Open Machete File** &mdash; open `machete` file in IntelliJ editor (to see what this editor can do see `Edit machete file` section)
+- ![](docs/left_bar_actions/fetch_all_remotes.png) **Fetch All Remotes** &mdash; equivalent to `git fetch --all` command
+- The most suitable actions (each is equivalent to one of the context menu actions) for a current branch that include one or more of:
+    - ![](docs/left_bar_actions/push.png) **Push Current Branch**
+    - ![](docs/left_bar_actions/pull.png) **Pull Current Branch**
+    - ![](docs/left_bar_actions/reset.png) **Reset Current Branch to Remote**
+    - ![](docs/left_bar_actions/slide_out.png) **Slide Out Current Branch**
+    - ![](docs/left_bar_actions/rebase.png) **Rebase Current Branch Onto Parent**
+    - ![](docs/left_bar_actions/override_forkpoint.png) **Override Fork Point of Current Branch**
+
+  Available action is selected based on a relation between a current branch and its parent and remote branch.
+- ![](docs/left_bar_actions/slide_in.png) **Slide In Branch Below Current Branch** &mdash; shortcut of `slide in` action for current branch
+- ![](docs/left_bar_actions/help.png) **Show Help Window** &mdash; show window with a sample branch layout and explanation what parts of this graph mean
+</details>
+
+<details>
+<summary><strong>Multi-repository support</strong></summary>
+<br/>
+
+Git Machete supports many git repositories in one project, including both regular repositories and submodules.
+If more than one repository is detected, the selection list will appear in the top of a main plugin window.
+In that case, each repository will have its own `machete` file.
+
+![](docs/multi_repo.gif)
+</details>
+
+## Build
+
+<details>
+<summary><strong>Prerequisites</strong></summary>
```

Comment:
Are we sure that we want to maintain the same content within 2 different files? This is redundant and easily leads to inconsistency.

Reply:
Ok, so I removed the content of this section and added a reference to DEVELOPMENT.md